build/chocolatey/jfrog-cli.nuspec

```diff
@@ -13,9 +13,9 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <bugTrackerUrl>https://github.com/jfrog/jfrog-cli/issues</bugTrackerUrl>
     <tags>JFrog CLI admin</tags>
-    <summary>JFrog CLI provides a simple interface that automates access to JFrog products: JFrog Artifactory, JFrog Mission Control, JFrog Bintray and JFrog Xray.</summary>
+    <summary>JFrog CLI provides a simple interface that automates access to JFrog products: JFrog Artifactory, JFrog Mission Control and JFrog Xray.</summary>

```

Comment:
<summary>JFrog CLI provides a simple interface that automates access to JFrog products: JFrog Artifactory, JFrog Mission Control and JFrog Xray.</summary>
-->
<summary>JFrog CLI provides a simple interface that automates access to JFrog Artifactory, Xray, Distribution and Mission Control.</summary>